docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,137 @@
+# Release 381 (13 May 2022)
+
+## General
+
+* Add support for table functions. ({issue}`1839`, {issue}`11937`)
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage . ({issue}`12211`)
+* Increase the default number of stages allowed for a query from 100 to 150,
+  specified with `query.max-stage-count`, so large queries do not fail. ({issue}`12292`)
+* Use the `fault_tolerant_execution_partition_count` session property, or the
+  `fault-tolerant-execution-partition-count` configuration property, to adjust
+  the number of partitions for distributed joins and aggregations executed with
+  fault-tolerant execution enabled (`retry-policy=TASK`). ({issue}`12263`)
+* Improve Advanced Encryption Standard (AES) processing performance on ARM64 processors. ({issue}`12251`)
```

Comment:
please confirm rewording @sopel39

Reply:
Except for the ARM64 docker image, this is not user visible (documentation change, plus product test changes).

For the ARM64 docker image, we should describe what, specifically, gets better from this change: "Improve performance when accessing S3 via SSL."

Reply:
Moved to the Docker image section and reworded